src/components/sbb-checkbox-group/sbb-checkbox-group.tsx

```diff
@@ -139,24 +123,29 @@ export class SbbCheckboxGroup implements ComponentInterface {
     }
   }
 
-  private get _checkboxes(): HTMLSbbCheckboxElement[] {
-    return (
-      Array.from(this._element.querySelectorAll('sbb-checkbox')) as HTMLSbbCheckboxElement[]
-    ).filter((el) => el.closest('sbb-checkbox-group') === this._element);
+  private get _checkboxes(): SbbCheckbox[] {
+    return (Array.from(this.querySelectorAll('sbb-checkbox')) as SbbCheckbox[]).filter(
```

Comment:
```suggestion
    return (Array.from(this.querySelectorAll('sbb-checkbox'))).filter(
```
Not 100% sure though